bin/peas_shell

```diff
@@ -49,18 +54,28 @@ class PanSensorShell(cmd.Cmd):
 
     def do_last_reading(self, device):
         """ Gets the last reading from the device. """
-        if hasattr(self, device):
-            print_info('*' * 80)
-            print("{}:".format(device.upper()))
-
-            rec = None
-            if device == 'weather':
-                rec = self.db.current.find_one({'type': 'weather'})
-            elif device == 'environment':
-                rec = self.db.current.find_one({'type': 'environment'})
-
-            pprint(rec)
-            print_info('*' * 80)
+        if not device:
+            print_warning('Usage: last_reading <device>')
+            return
+        if not hasattr(self, device):
+            print_warning('No such sensor: {!r}'.format(device))
+            return
+        print_info('*' * 80)
+        print("{}:".format(device.upper()))
+
+        rec = None
+        if device == 'weather':
+            rec = self.db.current.find_one({'type': 'weather'})
+        elif device == 'environment':
+            rec = self.db.current.find_one({'type': 'environment'})
+
```

Comment:
Since you are here, might be nice to pull the date from the `rec` and give a report of how stale it is. Just a `x seconds old` at bottom.